### ISSUE TITLE
ITSI redirects fail on staging

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#420]
Logging in via ITSI fails after entering credentials in the staging portal and being redirected back to the site.

![screenshot from 2015-06-16 16 47 26](https://cloud.githubusercontent.com/assets/1014341/8194107/6c026468-1447-11e5-98af-b00f447a3f70.png)

In addition to the error, the handling of the error produces 2 login modals overlapping each other.

**Comment by rajadain** [comment]
Can we check if the `MMW_ITSI_SECRET_KEY` variable is set in the staging environment? The double login is a genuine problem, will check that out.

**Comment by hectcastro** [comment]
I manually added the secret to staging, so you should be able to test this better now. Working on getting it added to AMIs in an automated fashion today.

**Comment by rajadain** [comment]
Thanks, that seems to have fixed it. Just created an account via ITSI:

![image](https://cloud.githubusercontent.com/assets/1430060/8213961/d6aaeb08-14f3-11e5-855e-780972234fc4.png)

@mmcfarland should I close this and make a new issue about the double login? Or just reword this to fit?

**Comment by mmcfarland** [comment]
Yeah, let's close this and have a dedicated issue.

**Comment by rajadain** [comment]
Created #442.
